README.md

```diff
@@ -128,6 +135,6 @@ Enjoy your Spark journey!
 
 For more information, please consult the [User Guide](docs/user_guide.md).
 
-[
+
```

Comment:
Can the Accessing the Jupyter Notebook section here be incorporated into the user guide?

Reply:
oh well, next time maybe. Not a big deal